src/MetaModels/DcGeneral/Events/Table/Attribute/Subscriber.php

```diff
@@ -519,24 +515,4 @@ public function handleUpdateAttribute(PostPersistModelEvent $event)
             }
         }
     }
-
-    /**
-     * Handle the deletion of an attribute and all attached data.
-     *
-     * @param PreDeleteModelEvent $event The event.
-     *
-     * @return void
-     */
-    public function handleDeleteAttribute(PreDeleteModelEvent $event)
-    {
-        if (($event->getEnvironment()->getDataDefinition()->getName() !== 'tl_metamodel_attribute')) {
-            return;
-        }
-
-        $attribute = $this->createAttributeInstance($event->getModel()->getPropertiesAsArray());
-
-        if ($attribute) {
-            $attribute->destroyAUX();
```

Comment:
may be we can find this at "deep delete"
https://github.com/contao-community-alliance/dc-general/pull/216/files#diff-4d17669c35dbb92b4821be484b80c92cR277